src/event_field_transformations.cpp

```diff
@@ -2,10 +2,18 @@
 
 #include <set>
 
+#include "mapdata.h"
 #include "mtype.h"
 #include "string_id.h"
 #include "type_id.h"
 
+static std::vector<cata_variant> mount_to_mounted( const cata_variant &v )
```

Comment:
This could be a generic `is_nonempty` that works on an arbitrary variant by examining the string directly.  That's probably a good idea, although it conflicts a little with (unpublished) changes I made to improve type safety.